clone_twitter/twitter/settings.py

```diff
@@ -36,13 +36,13 @@ def get_secret(setting, secrets=secrets):
         raise ImproperlyConfigured(error_msg)
 
 SECRET_KEY = get_secret("SECRET_KEY")
-
+FRONT_URL = get_secret("FRONT_URL")
```

Comment:
secrets.json needs update. (hiding front url is not required..but heard that front url will be changed so)